modules/test/playwright/tests/analytics-web/supportLiferayCookieManager.spec.ts

```diff
@@ -76,10 +76,13 @@ async function changeCookiePreference(page, options) {
 	await page.waitForTimeout(3000);
 }
 
-async function connectACToDXP(apiHelpers, page) {
-	const propertyName = 'My Property - ' + getRandomString();
+async function connectACToDXP(apiHelpers, page: Page) {
```

Comment:
@ViniLopes20 , can we define the type of `apiHelpers` parameter?

Reply:
Done